test/e2e/main_test.go

```diff
@@ -124,7 +133,26 @@ func testMainWrapper(m *testing.M) int {
 		df.Logger.Fatalf("Failed to create the CatalogSource custom resource using the %s index image: %v", indexImage, err)
 	}
 	if !df.RunDevSetup {
-		defer df.DeleteRegistryResources(registryProvisioned, catalogSourceName, catalogSourceNamespace)
+		var (
+			errArr []string
```

Comment:
```suggestion
			errors []string
```

Reply:
unless that's a keyword, idk reads easier to me. I can look and see the type so having in the name doesn't help me much.